packages/integration-tests/src/package_manager_sanity_checks.test.ts

```diff
@@ -81,13 +76,11 @@ void describe('getting started happy path', async () => {
   void it('creates new project and deploy them without an error', async () => {
     // TODO: remove the condition once GA https://github.com/aws-amplify/amplify-backend/issues/1013
     if (packageManager === 'yarn-classic') {
-      await execa('yarn', ['global', 'add', 'create-amplify@beta']);
-      process.env.npm_config_user_agent = 'yarn/1.22.21';
-      await runPackageManager(
-        packageManager,
-        ['create', 'amplify', '--yes'],
-        tempDir
-      ).run();
+      await execa('yarn', ['add', 'create-amplify@beta'], { cwd: tempDir });
+      await execaCommand('./node_modules/.bin/create-amplify --yes --debug', {
```

Comment:
could we run `create amplify@beta --yes` here instead of reaching into the bin folder?

Reply:
no, because 1) `create amplify` uses global installed package from `~/.config/yarn/global/`, 2) yarn-classic `create` doesn't accept tags 😂  https://github.com/yarnpkg/yarn/issues/6587